save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -364,6 +370,13 @@ class VulnerabilityService(
         lnkVulnerabilityUserRepository.deleteByUserNameAndVulnerabilityId(userName, vulnerability.requiredId())
     }
 
+    /**
+     * @param vulnerabilityId id of vulnerability
+     * @return list of LnkVulnerabilityUser link to vulnerability
+     */
+    @Transactional
```

Comment:
maybe add reaonly=true?

Reply:
Yes, I removed that annotation. The class itself is already marked ReadOnly